src/styles/index.scss

```diff
@@ -1,3 +1,148 @@
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+
 body {
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
+  align-items: center;
+  width: 200px;
+  background-color: white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
+
+  &__image {
+    box-sizing: border-box;
+    background-image: url(../images/imac.jpeg);
+    width: 160px;
+    height: 134px;
+    background-size: cover;
+    margin: 0 auto;
+  }
+
+  &__product-name {
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: #{$main-accent};
+    width: 100%;
+    margin-top: 40px;
+    margin-bottom: 0;
+  }
+
+  &__product-id {
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    color: #{$secondary};
+    margin-top: 4px;
+  }
+
+  &__reviews {
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+    margin-top: 16px;
+  }
+
+  &__stars {
+    display: flex;
+    margin: 0;
+  }
+
+  &__star {
+    background-image: url(../images/star.svg);
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
```

Comment:
The number of active stars should depend on the class with the modifier. Consider using classes from the previous task for styling